src/pandapipes/pf/result_extraction.py

```diff
@@ -62,7 +62,8 @@ def get_basic_branch_results(net, branch_pit, node_pit):
     t1 = node_pit[to_nodes, TINIT_NODE]
     vf = branch_pit[:, MDOTINIT] / get_fluid(net).get_density(NORMAL_TEMPERATURE)
     v = branch_pit[:, MDOTINIT] / fluid.get_density(NORMAL_TEMPERATURE) / branch_pit[:, AREA]
-    return v, branch_pit[:, MDOTINIT], vf, from_nodes, to_nodes, t0, t1, branch_pit[:, RE], \
+    t_outlet = branch_pit[:, TOUTINIT]
```

Comment:
Since there are so many different values extracted here, we might just move the dictionary creation to this function, instead of returning so many values and then just making a dictionary in the calling function.